tests/unit/amo/sagas/test_users.js

```diff
@@ -96,6 +98,21 @@ describe(__filename, () => {
           expect(error).toBe(expectedError);
         });
     });
+
+    it('handles 401 responses', async () => {
+      const expectedError = createApiError({ response: { status: 401 } });
```

Comment:
It might be worth having another test which also has an error response, but not a `401` to verify that it is re-thrown.